test/integration/default/inspec/default_spec.rb

```diff
@@ -0,0 +1,6 @@
+describe file('/etc/resolv.conf') do
+  its('content') { should match /^search localdomain/ }
+  its('content') { should match /^nameserver 1\.1\.1\.1/ }
+  its('content') { should match /^nameserver 8\.8\.8\.8/ }
+  its('content') { should match /^options rotate timeout:2/ }
+end
```

Comment:
I think it might be useful to actually test that DNS resolution is working after making these changes. A simple test would be to do a `ping -c 1 google.com` command that returns 0.